From f4d05174b4bbcde262c4eab1d316c4725957da42 Mon Sep 17 00:00:00 2001
From: LewiVir <lewivir@163.com>
Date: Thu, 17 Dec 2020 12:11:35 +0800
Subject: [PATCH 1/2] Revert rockchip uboot to 2020.07

---
 package/boot/uboot-rockchip/Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/package/boot/uboot-rockchip/Makefile b/package/boot/uboot-rockchip/Makefile
index 3309912cf..b2333cb99 100644
--- a/package/boot/uboot-rockchip/Makefile
+++ b/package/boot/uboot-rockchip/Makefile
@@ -5,10 +5,10 @@
 include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
-PKG_VERSION:=2020.10
-PKG_RELEASE:=1
+PKG_VERSION:=2020.07
+PKG_RELEASE:=3
 
-PKG_HASH:=0d481bbdc05c0ee74908ec2f56a6daa53166cc6a78a0e4fac2ac5d025770a622
+PKG_HASH:=c1f5bf9ee6bb6e648edbf19ce2ca9452f614b08a9f886f1a566aa42e8cf05f6a
 
 PKG_MAINTAINER:=Tobias Maedel <openwrt@tbspace.de>
 
@@ -103,4 +103,4 @@ endef
 define Package/u-boot/install/default
 endef
 
-$(eval $(call BuildPackage/U-Boot))
\ No newline at end of file
+$(eval $(call BuildPackage/U-Boot))
-- 
2.25.1

